decidim-participatory_processes/app/controllers/decidim/participatory_processes/participatory_process_groups_controller.rb

```diff
@@ -15,7 +15,11 @@ def show
       private
 
       def participatory_processes
-        @participatory_processes ||= group.participatory_processes.published
+        @participatory_processes ||= if current_user
+                                       group.participatory_processes.private_spaces_user(current_user.id).published
```

Comment:
Maybe we could use the `PrivateParticipatoryProcesses` query here instead? It looks like it's mostly doing the same thing.

Reply:
We are querin the processes of a group of processes... I don't thins that is the same..

Reply:
Oh sorry! My fault.